pkg/config/config_test.go

```diff
@@ -190,3 +192,20 @@ func TestGetFromSecrets(t *testing.T) {
 		t.Log(err)
 	}
 }
+
+func TestAccessTokenRetrieval(t *testing.T) {
+	reader := setup(t)
+	cfg, _ := ReadCRSpecFromFile(reader)
+	if cfg.Spec.Git.SecretName != "" {
+		// skipped because need kubectl (will not perform ci checks)
+		// if need to test, comment line bellow and add secretName to example spec
+		t.Skip()
+	} else {
+		cfg.Spec.AddToSecrets("qliksense2", "mongo", "tadadaa", "")
+		if _, err := cfg.Spec.Git.GetAccessToken(); err != nil {
```

Comment:
in your test create the secrets kubectl first

Reply:
created secret